src/main/java/uk/gov/justice/digital/delius/controller/secure/AppointmentBookingController.java

```diff
@@ -71,4 +73,25 @@
         AppointmentCreateResponse response = appointmentService.createAppointment(crn, sentenceId, contextName, contextlessAppointmentCreateRequest);
         return new ResponseEntity<>(response, HttpStatus.CREATED);
     }
+
+    @RequestMapping(value = "/offenders/crn/{crn}/appointments/{appointmentId}/context/{contextName}",
+        method = RequestMethod.PATCH,
+        consumes = "application/json")
+    @ApiResponses(
+        value = {
+            @ApiResponse(code = 200, message = "Updated", response = String.class),
+            @ApiResponse(code = 400, message = "Invalid request", response = ErrorResponse.class),
+            @ApiResponse(code = 403, message = "Requires role ROLE_COMMUNITY_INTERVENTIONS_UPDATE"),
+            @ApiResponse(code = 500, message = "Unrecoverable error whilst processing request.", response = ErrorResponse.class)
+        })
+
+    @ApiOperation(value = "Updates an Contact appointment")
+    public ResponseEntity<AppointmentUpdateResponse> patchAppointmentWithContext(final @PathVariable("crn") String crn,
+                                                                                 final @PathVariable("appointmentId") Long appointmentId,
+                                                                                 final @PathVariable("contextName") String context,
+                                                                                 final @RequestBody JsonPatch jsonPatch) {
+
+        AppointmentUpdateResponse response = appointmentService.patchAppointment(crn, appointmentId, context, jsonPatch);
+        return new ResponseEntity<>(response, HttpStatus.OK);
```

Comment:
You could yes make this function return `AppointmentUpdateResponse` - no need for the ResponseEntity

Reply:
Will do